controller-podmonitor.jsonnet

```
@@ -1,9 +1,9 @@
-// Prometheus Service Monitor manifest
+// Prometheus Pod Monitor manifest
```

Add reference to PodMonitor def
```suggestion
// Prometheus Pod Monitor manifest
// ref: https://github.com/prometheus-operator/prometheus-operator#customresourcedefinitions
```